### ISSUE TITLE
storaged incorrectly reports a filesystem inconsistency

### ISSUE THREAD

**Issue by ananthb** [cockpit-project/cockpit#18654]
### Explain what happens

Cockpit storaged incorrectly assumes that my EFI partition will not be mounted automatically at boot.

1. Create an EFI partition with UUID c12a7328-f81f-11d2-ba4b-00a0c93ec93b following the [Discoverable Partitions Specification](https://uapi-group.org/specifications/specs/discoverable_partitions_specification/).
2. Arch Linux Systemd automounts partitions following the Discoverable Partitions Specification via [systemd-gpt-auto-generator](https://man.archlinux.org/man/systemd-gpt-auto-generator.8).
3. Cockpit storaged incorrectly reports that the EFI partition will not be mounted automatically.

<img width="1232" alt="Screenshot 2023-04-18 at 02 32 25" src="https://user-images.githubusercontent.com/3133350/232610850-52966bd5-872a-45fc-a742-e54e6b4078b1.png">

### Version of Cockpit

289

### Where is the problem in Cockpit?

Storage

### Server operating system

Arch Linux

### Server operating system version

latest

### What browsers are you using?

Firefox, Firefox Mobile on Android, Chrome

**Comment by martinpitt** [comment]
Right, the storage page should probably ignore these discovered mount units. Thanks for the report!
